Dockerfile

```diff
@@ -117,6 +118,20 @@ RUN echo '{ \
     \n      "debian:stretch" \
     \n    ], \
     \n    [ \
+    \n      "dcm2niix", \
+    \n      { \
+    \n        "version": "v1.0.20171215" \
```

Comment:
Hm... This one is old , better use more recent one, there were various fixes which are relevant

Reply:
it is labelled as a pre-release so I was hesitant, but if you are vouching i'm for adding the latest and greatest